admin/controllers/pupil.js

```diff
@@ -257,6 +261,28 @@ const postEditPupil = async (req, res, next) => {
   res.redirect(`/school/pupil-register/lastName/true?hl=${pupilId}`)
 }
 
+const getPrintPins = async (req, res, next) => {
+  res.locals.pageTitle = 'Print pupils'
+  let pupils
+  let school
+  let qrDataURL
+  const date = dateService.formatDayAndDate(new Date())
+  try {
+    pupils = await generatePinsService.getPupilsWithActivePins(req.user.School)
+    school = await generatePinsService.getActiveSchool(req.user.School)
```

Comment:
Surprising.  Is their logic in this call particular to generating pins? I would expect the call to be to the SchoolService.

Reply:
I think I should rename this to getSchoolWithActivePass

Reply:
well its relevant to fetching an active pass that's why I kept it in this service.

Reply:
Perhaps it's better for it to lie under school service yea

Reply:
if it's returning the school pin, then rename the method to something more appropriate like 'getSchoolPin'.

The current name doesn't make any sense in this context anyway